Detectors/AOD/src/AODProducerWorkflowSpec.cxx

```diff
@@ -2593,7 +2596,7 @@ DataProcessorSpec getAODProducerWorkflowSpec(GID::mask_t src, bool enableSV, boo
   outputs.emplace_back(OutputLabel{"O2mcparticle_001"}, "AOD", "MCPARTICLE_001", 0, Lifetime::Timeframe);
   outputs.emplace_back(OutputLabel{"O2mctracklabel"}, "AOD", "MCTRACKLABEL", 0, Lifetime::Timeframe);
   outputs.emplace_back(OutputLabel{"O2mfttrack"}, "AOD", "MFTTRACK", 0, Lifetime::Timeframe);
-  outputs.emplace_back(OutputLabel{"O2track_iu"}, "AOD", "TRACK_IU", 0, Lifetime::Timeframe);
+  outputs.emplace_back(OutputLabel{"Tracks_IU"}, "AOD", "TRACK_IU", 0, Lifetime::Timeframe);
```

Comment:
One should probably use `o2::aod::MetadataTrait<o2::aod::StoredTracksIU>::metadata::tableLabel()` and the the various `o2::aod::MetadataTrait<o2::aod::StoredTracksIU>::metadata::origin()` `o2::aod::MetadataTrait<o2::aod::StoredTracksIU>::metadata::description()`.